crates/modelardb_server/src/storage/mod.rs

```diff
@@ -239,8 +237,32 @@ impl StorageEngine {
         })
     }
 
-    /// Add references to the [`uncompressed_data_buffer::UncompressedDataBuffer`] currently on disk
-    /// to [`UncompressedDataManager`].
+    /// Start `threads` threads with `name` that executes `function` and whose [`JoinHandle`] is
+    /// added to `join_handles.
+    fn start_threads<F>(
+        threads: usize,
```

Comment:
Maybe rename this to `num_threads` or something like that to make it clear it is the number of threads that are started and not the actual threads.